node/indexer/indexer_test.go

```diff
@@ -34,6 +34,10 @@ import (
 	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
+const genesisID string = "foo"
+
+var genesisHash crypto.Digest
```

Comment:
rename these to testGenesisID and testGenesisHash.
The testGenesisHash can be defined here as 
```golang
var testGenesisHash = crypto.Digest{0x1, 0x2, 0x3..}
```
which is abit cleaner than defining it here and initializing it in `init`

Reply:
done